File: src/targets/clojure/clj_http/fixtures/multipart-file.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/post "https://httpbin.org/anything" {:multipart [{:name "foo"
                                                          :content (clojure.java.io/file "src/fixtures/files/hello.txt")}]})
=======
(client/post "http://mockbin.com/har" {:multipart [{:name "foo"
                                                    :content (clojure.java.io/file "test/fixtures/files/hello.txt")}]})
>>>>>>> upstream/master

File: src/targets/php/guzzle/fixtures/multipart-file.php
<?php
<<<<<<< HEAD
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'multipart' => [
    [
        'name' => 'foo',
        'filename' => 'src/fixtures/files/hello.txt',
=======

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
  'multipart' => [
    [
        'name' => 'foo',
        'filename' => 'test/fixtures/files/hello.txt',
>>>>>>> upstream/master
        'contents' => null,
        'headers' => [
                'Content-Type' => 'text/plain'
        ]
    ]
  ]
]);

echo $response->getBody();